api/server/server.go

```diff
@@ -899,14 +899,27 @@ func (s *Server) getImagesGet(eng *engine.Engine, version version.Version, w htt
 	if version.GreaterThan("1.0") {
 		w.Header().Set("Content-Type", "application/x-tar")
 	}
-	var job *engine.Job
+	output := utils.NewWriteFlusher(w)
+
+	imageExportConfig := &graph.ImageExportConfig{
+		Engine:    eng,
+		Outstream: output,
+	}
 	if name, ok := vars["name"]; ok {
-		job = eng.Job("image_export", name)
+		imageExportConfig.Name = append(imageExportConfig.Name, name)
```

Comment:
@duglin  U R right here definitely. Just a habit. Will fix it in a min.
@LK4D4 i dont mind if you prefer other term. What about `ImageName`?